src/views/Dashboard/View.vue

```diff
@@ -75,7 +75,7 @@ export default {
       delete localStorage.content
       console.log(localStorage.content)
     } else {
-      fetch("http://localhost:3000/:id/View/:id")
+      fetch("https://speedwagonmailback.herokuapp.com/:id/View/:id")
           .then(response => response.json())
```

Comment:
ini juga linknya samain sama si elvin bikin 

https://github.com/A-WebProgramming-Kelompok-9/EmailBackend/blob/master/index.js

Reply:
app.post("/email/find", (req, res) => {
    emaildb.getEmail(req.body.id).then((ress, err) => {
        res.send(ress)
    });
})

Reply:
yang ini id nya sementara kosongin dlu juga gapapa, tapi siapin dlu aja bentuk fetchnya 
kira2 bentuknya kek gini 

fetch("https://speedwagonmailback.herokuapp.com/account/add", {
        method: "POST",
        body: JSON.stringify({
          usern: this.Username,
          pass: this.Password,
          altermail: this.Alt_Email
        }),
        headers: {
          "content-type": "application/json"
        }
      }).then(response => response.json()
      ).then(result => {
        console.log(result)
      })